person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..100)
     @age = age
     @name = name
```

Comment:
- According to the project requirements, this class should have a `correct_name ` method that returns the name attribute 🙏

Reply:
Thank you, I have worked on the changes.